addons/website_sale/controllers/main.py

```diff
@@ -1,17 +1,15 @@
 # -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
 import werkzeug
 
-from openerp import SUPERUSER_ID
-from openerp import http
-from openerp import tools
-from openerp.http import request
-from openerp.tools.translate import _
-from openerp.addons.website.models.website import slug
+from odoo import http, tools, _
+from odoo.http import request
+from odoo.addons.website.models.website import slug
 
-PPG = 20 # Products Per Page
-PPR = 4  # Products Per Row
+PPG = 20  # Products Per Page
+PPR = 4   # Products Per Row
```

Comment:
if those lines are going to get rewritten anyway, maybe those things could be renamed to ProductsPerPage and ProductsPerRow or something?